plugins/not-found/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@backstage/plugin-not-found",
+  "version": "0.1.1-alpha.20",
```

Comment:
I think the version numbers should all be set to the same as every other plugin.

Reply:
Thanks, updated the version to `0.1.1-alpha.21`